en/docs/tutorials/scenarios/scenario9-realtime-data.md

```diff
@@ -1,4 +1,44 @@
 # Scenario 9 - Realtime Data with WebSocket API
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Quantis wants to provide real time location of their trains to their customers. The sensors in the train will 
+be providing real time events and Quantis wants to convert it to websockets and expose it as a Streaming API 
+so that the client mobile applications can subscribe to it and receive real time events.**
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/scenario_real_time_notifications.png" alt="Realtime Notifications" title="Realtime Notifications" width="60%" />
+
+WSO2 Streaming Integrator(SI) is a streaming data processing server that integrates streaming data and takes 
+action based on streaming data. Quantis is planning to use Streaming Integrator to process the real time event data.
+
+To develop a service in Streaming Integrator, you need to use Streaming Integrator tooling. Siddhi is the 
+Event Processing Engine that is used inside the Streaming Integrator. To process the websockets you can write 
+the service in SIddhi Query Language. You can test the service in the tooling itself. Once the development is 
+complete you can export it as a Siddhi app and add it to the Streaming Integrator runtime.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/streaming_api_tooling.png" alt="Streaming Integrator Tooling" title="Streaming Integrator Tooling" width="60%" />
+
+Here, for simplicity, the service is already created and exported as a SiddhiApp and added to the Streaming Integrator instance in the tutorial setup. You can test the Streaming Integrator service by invoking it directly. You will start receiving Realtime events by connecting a websocket client. (You can use [wscat](https://www.npmjs.com/package/wscat))
+
+```
+wscat -c ws://localhost:8025/ 
+
+```
+
+Once you expose the events via a Websocket Server, you can expose the websockets with API Manager as you do with other APIs where you can provide **secure access, Rate limiting, Throttling, Monetization, Analytics** etc. The API is already published to the portal. To invoke the API, you can follow the below steps.
+
+1. To subscribe to the websocket API, go to [https://localhost:9443/devportal/](https://localhost:9443/devportal/) Dev portal and select **Quantis** tenant domain. (Logout from your previous tenant) This will redirect you to the Quantis developer portal.
```

Comment:
```suggestion
1. To subscribe to the websocket API, go to [https://localhost:9443/devportal/](https://localhost:9443/devportal/) Developer Portal and select **Quantis** tenant domain. (You need to log out from your previous tenant). This will redirect you to the Quantis Developer Portal.
```